Sample.java

```diff
@@ -0,0 +1,5 @@
+class Sample{
+    public static void main(String[] args) {
+        System.out.println("Hello, World.");
+    }
+}
```

Comment:
nice 🎉 
Make sure you always have an empty line as the last line of your file going forward